Source/Lib/Codec/EbCodingLoop.c

```diff
@@ -2272,6 +2272,13 @@ void SetPmEncDecMode(
 
         }
     }
+
+    if (pictureControlSetPtr->ParentPcsPtr->segmentOvArray != NULL && sequenceControlSetPtr->staticConfig.segmentOvEnabled) {
+		SegmentOverride_t* segmentOvPtr = pictureControlSetPtr->ParentPcsPtr->segmentOvArray;
+        if(segmentOvPtr[lcuIndex].ovFlags & EB_TU_FILTER_OV)
+            contextPtr->pmpMaskingLevelEncDec= CLIP3(0, 7, contextPtr->pmpMaskingLevelEncDec + segmentOvPtr[lcuIndex].filterOv);
```

Comment:
A space should be added before "=". And similiar coding style flaws in other changes.